### ISSUE TITLE
[FrontendV2] Incorrect success message on page access

### ISSUE THREAD

**Issue by ankitpal1029** [Cloud-CV/EvalAI#3376]
![image](https://user-images.githubusercontent.com/49815751/112798200-355b6200-908a-11eb-97a2-fbd7fbf7aa28.png)

i have set up the docker image but every time i visit a new route i get a success message with an X right next to it .This happens when logged in as a new user who's not authenticated their email id.

OS: Linux (PopOs 20.10)
Browser: Brave browser

**Comment by niharsalunke** [comment]
@Ram81 may I work on this?

**Comment by niharsalunke** [comment]
@ankitpal1029 are you working on this already?

**Comment by Ram81** [comment]
@niharsalunke please complete #3346 before starting on another issue

**Comment by niharsalunke** [comment]
Sure @Ram81. Working on it.

**Comment by ankitpal1029** [comment]
@Ram81 it's to be noted that this error occurs only with a newly created user and not the default 3 users already provided

**Comment by ankitpal1029** [comment]
> @ankitpal1029 are you working on this already?

@niharsalunke yes i'm working on it currently.

**Comment by ankitpal1029** [comment]
@Ram81  i think this is an issue of email verification, for context frontend version 1 doesn't allow the user to access the challenges till they have verified their email and instead shows them this page,
![image](https://user-images.githubusercontent.com/49815751/112803767-46f43800-9091-11eb-9551-83a1cd67cf2c.png)
note this is after selecting a challenge and clicking on it.
Maybe we should implement something of this sort for v2 as well, what are your views on this?

**Comment by ankitpal1029** [comment]
@Ram81 i think i have figured out the cause of the issue , 
on recieving a 403 error the global.service is trying to access err.error['error'] whereas
![image](https://user-images.githubusercontent.com/49815751/112862127-12a16b80-90d3-11eb-9c9c-287205b343f9.png)
 it recieves an error with field detail:
![image](https://user-images.githubusercontent.com/49815751/112862029-fd2c4180-90d2-11eb-8fd7-07c2c4681c51.png)
fixing that gives us the appropriate error message: 
![image](https://user-images.githubusercontent.com/49815751/112862276-3c5a9280-90d3-11eb-8985-f96382a3f633.png)

**Comment by ankitpal1029** [comment]
@Ram81 can you take a look at my pr

**Comment by ankitpal1029** [comment]
@Ram81 can you take a look at the pr

**Comment by AaqibAhamed** [comment]
PR Raised

**Comment by RishabhJain2018** [comment]
Closing this since we are not working on Frontend_v2
